eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -749,6 +749,33 @@
           "code" : "java.method.removed",
           "old" : "method com.azure.resourcemanager.containerservice.models.ManagedClusterNatGatewayProfile com.azure.resourcemanager.containerservice.models.ManagedClusterNatGatewayProfile::withEffectiveOutboundIPs(java.util.List<com.azure.resourcemanager.containerservice.models.ResourceReference>)",
           "justification": "The effectiveOutboundIPs was never a client-side modifiable property."
+        },
+        {
+          "regex": true,
+          "ignore": true,
+          "code": "java.class.nowFinal",
+          "new": "class com\\.azure\\.resourcemanager\\.eventgrid\\.models\\.(DeadLetterWithResourceIdentity|DeliveryWithResourceIdentity|EventSubscriptionIdentity)",
+          "justification": "Class is now final."
+        },
+        {
+          "ignore": true,
+          "code" : "java.method.returnTypeTypeParametersChanged",
+          "old" : "method java.util.List<com.azure.resourcemanager.eventgrid.models.TopicTypePropertiesSupportedScopesForSourceItem> com.azure.resourcemanager.eventgrid.models.TopicTypeInfo::supportedScopesForSource()",
+          "new" : "method java.util.List<com.azure.resourcemanager.eventgrid.models.TopicTypeSourceScope> com.azure.resourcemanager.eventgrid.models.TopicTypeInfo::supportedScopesForSource()",
+          "justification": "The return type changed to 'java.util.List<com.azure.resourcemanager.eventgrid.models.TopicTypeSourceScope>'."
```

Comment:
```suggestion
          "justification": "Service swagger fix. Anonymous enum being named."
```

Reply:
fixed by [e710bfd](https://github.com/Azure/azure-sdk-for-java/pull/44748/commits/e710bfd79dd10e3f5ff32f27b313005ab17333a4)